dashboard/admin.py

```diff
@@ -25,7 +37,7 @@ class CourseViewOptionInline(admin.StackedInline):
     model = CourseViewOption
     form = AlwaysChangedModelForm
 
-    exclude = ()
+    exclude: Tuple[str, ...] = ()
```

Comment:
Does this need an annotation to fix something? It's not a return value or a parameter...

Reply:
Yes this was needed to allow both the empty and defined string. Without this it mypy returns.

`dashboard/admin.py:45: error: Incompatible types in assignment (expression has type "Tuple[str]", variable has type "Tuple[]")`

This could have been avoided by assigning some initial value, but it was the only workaround I could find for this.